### PR TITLE
replaced invalid links to new links

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #4801 , so i replaced invalid links to  new links.
below are the new links

**https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-1-8211-introduction-to-the-sysvol/ba-p/423456

https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-2-8211-dfsrmig-exe-the-sysvol/ba-p/423470

https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-3-migrating-to-the-prepared-state/ba-p/423503

https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-4-8211-migrating-to-the-8216/ba-p/423514

https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-5-8211-migrating-to-the-8216/ba-p/423516

https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc732863(v=ws.10)

https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2003/cc759297(v=ws.10)

### CODE REVIEW

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md [hunk 1]
```diff
@@ -63,16 +63,16 @@ This document assumes that you have a basic knowledge of Active Directory Domain
...
+[SYSVOL Migration Series: Part 3—Migrating to the 'PREPARED' state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-3-migrating-to-the-prepared-state/ba-p/423503)
 
-[SYSVOL Migration Series: Part 4—Migrating to the 'REDIRECTED' state](https://go.microsoft.com/fwlink/?linkid=121759)
+[SYSVOL Migration Series: Part 4—Migrating to the 'REDIRECTED' state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-4-8211-migrating-to-the-8216/ba-p/423514)
 
-[SYSVOL Migration Series: Part 5—Migrating to the 'ELIMINATED' state](https://go.microsoft.com/fwlink/?linkid=121760)
+[SYSVOL Migration Series: Part 5—Migrating to the 'ELIMINATED' state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-5-8211-migrating-to-the-8216/ba-p/423516)
 
-[Step-by-Step Guide for Distributed File Systems in Windows Server 2008](https://go.microsoft.com/fwlink/?linkid=85231)
+[Distributed File Systems Step-By-Step Guide for Windows Server 2008](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc732863(v=ws.10))
```
Comment (illfated) [inline WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:76]: The first 5 links are fine, but we need to use language-specific links for the last 2:
```suggestion
[DFS Step-by-Step Guide for Windows Server 2008](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc732863(v=ws.10))
```
(also copying in the exact page title in the descriptive link text)

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md [hunk 2]
```diff
@@ -63,16 +63,16 @@ This document assumes that you have a basic knowledge of Active Directory Domain
...
 
-[FRS Technical Reference](https://go.microsoft.com/fwlink/?linkid=121764)
+[FRS Technical Reference](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2003/cc759297(v=ws.10))
```
Comment (illfated) [inline WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:78]: (same as above)
```suggestion
[FRS Technical Reference](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2003/cc759297(v=ws.10))
```
The reason for including the language /en-us/ in these last 2 links is that those 2 pages are archived under /previous-versions/ and are therefore not part of the automated translation feature in the current docs.

**Comment by illfated** [comment]
@RAJU2529 : Your PR looks good so far, so if you would be so kind to merge my suggestions, it should be ready for final review.

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md (hunk 1 above)
Comment (illfated) [reply WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:76]: Hmm, odd. When I click the links in your PR comment, it looks like they do have the language-agnostic feature working. Need to check what is different between the comment links and the PR links. (Might need to cancel my suggestions.)

Comment (illfated) [reply WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:76]: Sorry, my bad. I have got an add-on in my browser, and that add-on links the URL only up until the (v=ws.10) part, which it does not include.

I will go ahead and delete these suggestions now and review again from a different browser.

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md [hunk 3]
```diff
@@ -63,16 +63,16 @@ This document assumes that you have a basic knowledge of Active Directory Domain
 
 ## Additional references
 
-[SYSVOL Migration Series: Part 1—Introduction to the SYSVOL migration process](https://go.microsoft.com/fwlink/?linkid=121756)
+[SYSVOL Migration Series: Part 1—Introduction to the SYSVOL migration process](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-1-8211-introduction-to-the-sysvol/ba-p/423456)
```
Comment (illfated) [inline WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:66]:
```suggestion
[SYSVOL Migration Series: Part 1 – Introduction to the SYSVOL migration process](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-1-8211-introduction-to-the-sysvol/ba-p/423456)
```
(cosmetic text adjustment)

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md [hunk 4]
```diff
@@ -63,16 +63,16 @@ This document assumes that you have a basic knowledge of Active Directory Domain
...
 
-[SYSVOL Migration Series: Part 2—Dfsrmig.exe: The SYSVOL migration tool](https://go.microsoft.com/fwlink/?linkid=121757)
+[SYSVOL Migration Series: Part 2—Dfsrmig.exe: The SYSVOL migration tool](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-2-8211-dfsrmig-exe-the-sysvol/ba-p/423470)
```
Comment (illfated) [inline WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:68]:
```suggestion
[SYSVOL Migration Series: Part 2 – Dfsrmig.exe: The SYSVOL migration tool](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-2-8211-dfsrmig-exe-the-sysvol/ba-p/423470)
```
(cosmetic text adjustment)

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md [hunk 5]
```diff
@@ -63,16 +63,16 @@ This document assumes that you have a basic knowledge of Active Directory Domain
...
 
-[SYSVOL Migration Series: Part 3—Migrating to the 'PREPARED' state](https://go.microsoft.com/fwlink/?linkid=121758)
+[SYSVOL Migration Series: Part 3—Migrating to the 'PREPARED' state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-3-migrating-to-the-prepared-state/ba-p/423503)
```
Comment (illfated) [inline WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:70]:
```suggestion
[SYSVOL Migration Series: Part 3 - Migrating to the 'PREPARED' state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-3-migrating-to-the-prepared-state/ba-p/423503)
```
(cosmetic text adjustment)

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md [hunk 6]
```diff
@@ -63,16 +63,16 @@ This document assumes that you have a basic knowledge of Active Directory Domain
...
 
-[SYSVOL Migration Series: Part 4—Migrating to the 'REDIRECTED' state](https://go.microsoft.com/fwlink/?linkid=121759)
+[SYSVOL Migration Series: Part 4—Migrating to the 'REDIRECTED' state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-4-8211-migrating-to-the-8216/ba-p/423514)
```
Comment (illfated) [inline WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:72]:
```suggestion
[SYSVOL Migration Series: Part 4 – Migrating to the ‘REDIRECTED’ state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-4-8211-migrating-to-the-8216/ba-p/423514)
```
(cosmetic text adjustment)

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md [hunk 7]
```diff
@@ -63,16 +63,16 @@ This document assumes that you have a basic knowledge of Active Directory Domain
...
 
-[SYSVOL Migration Series: Part 5—Migrating to the 'ELIMINATED' state](https://go.microsoft.com/fwlink/?linkid=121760)
+[SYSVOL Migration Series: Part 5—Migrating to the 'ELIMINATED' state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-5-8211-migrating-to-the-8216/ba-p/423516)
```
Comment (illfated) [inline WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:74]:
```suggestion
[SYSVOL Migration Series: Part 5 – Migrating to the 'ELIMINATED' state](https://techcommunity.microsoft.com/t5/storage-at-microsoft/sysvol-migration-series-part-5-8211-migrating-to-the-8216/ba-p/423516)
```
(cosmetic text adjustment)

File: WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md (hunk 1 above)
Comment (illfated) [inline WindowsServerDocs/storage/dfs-replication/migrate-sysvol-to-dfsr.md:76]:
```suggestion
[Distributed File Systems Step-by-Step Guide for Windows Server 2008](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc732863(v=ws.10))
```
(cosmetic text adjustment)

**Review by illfated — COMMENTED** [review]
Looks very good, so if we could just add the same spacing as in the page titles, it will be 100%.

**Comment by eross-msft** [comment]
@RAJU2529  Thank you for these updates. I will merge them now.

**Pull request merged**